pom.xml

```diff
@@ -33,7 +33,7 @@
   </parent>
 
   <artifactId>metrics</artifactId>
-  <version>4.0.2.0-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```

Comment:
As I said before , this is incompatible with using the release plugin.

I did a local `mvn release:prepare -DpushChanges=false` and this is what I got:

```
$ git log -n 3
commit 6bd1162977ff30464f5cef32b4859cbdfea0e2f3 (HEAD -> batmat-JENKINS-51308)
Author: Stephen Connolly <stephen.alan.connolly@gmail.com>
Date:   Thu Jun 21 15:16:16 2018 +0100

    [maven-release-plugin] prepare for next development iteration

commit f2ff01e30b97fbca38c649ee738bd20590f55d10 (tag: metrics-4.0.2.2)
Author: Stephen Connolly <stephen.alan.connolly@gmail.com>
Date:   Thu Jun 21 15:16:16 2018 +0100

    [maven-release-plugin] prepare release metrics-4.0.2.2

commit 5f86aa4fe63f3e2454abbac199567ae7fed7e585
Merge: 34c6bea 21b416d
Author: Stephen Connolly <stephen.alan.connolly@gmail.com>
Date:   Thu Jun 21 15:13:10 2018 +0100

    Merge branch 'JENKINS-51308' of https://github.com/batmat/metrics-plugin into batmat-JENKINS-51308

$ git diff HEAD^^
diff --git a/pom.xml b/pom.xml
index 9c3dad0..90a977f 100644
--- a/pom.xml
+++ b/pom.xml
@@ -33,7 +33,7 @@
   </parent>
 
   <artifactId>metrics</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>4.0.2.3-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
   <name>Metrics Plugin</name>
```

So *every* time somebody does a real release this change will be reverted by the release plugin.

I am not merging a PR for Incrementals if is blocks cutting actual releases or if it adds manual steps to the release process.

Reply:
@stephenc Yes you have to manually run `mvn incrementals:reincrementalify` after doing a release if you want to reenable incremental release ([Documentation](https://github.com/jenkinsci/incrementals-tools#running-maven-releases)). There is a ticket to do it automatically as part of the release process [JENKINS-51658](https://issues.jenkins-ci.org/browse/JENKINS-51658).